resources/definitions/fdmprinter.def.json

```diff
@@ -2569,6 +2599,20 @@
                             "limit_to_extruder": "top_bottom_extruder_nr",
                             "settable_per_mesh": true
                         },
+                        "jerk_sanding":
+                        {
+                            "label": "Sanding Jerk",
+                            "description": "The maximum instantaneous velocity change while performing sanding.",
```

Comment:
The description needs to change; In some places you call it sanding (here) and in other places you call it ironing. I think you need to pick one and stick with it. 
I'd also prefer it if the setting ids would also be consistent, but that is not as important.